clients/vault/src/oracle/agent.rs

```diff
@@ -77,14 +80,11 @@ pub async fn start_oracle_agent(
 	let collector_clone = collector.clone();
 
 	let shutdown_sender_clone = shutdown_sender.clone();
-	// a clone used to forcefully call a shutdown, when StellarOverlay disconnects.
-	let shutdown_sender_clone2 = shutdown_sender.clone();
-
 	// disconnect signal sender tells the StellarOverlayConnection to close its TcpStream to Stellar
 	// Node
 	let (disconnect_signal_sender, mut disconnect_signal_receiver) = mpsc::channel::<()>(2);
 
-	service::spawn_cancelable(shutdown_sender_clone.subscribe(), async move {
```

Comment:
One less subscription, and it's either here or line 130 that's causing the 
`Waiting for 1 tasks to shut down`.

Reply:
This will also look better on _`tokio-console`_, as it won't point to the `spawn_cancelable` line of code